src/transformStateWithClones.js

```diff
@@ -7,7 +7,31 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const copyState = { ...state };
+  const result = [];
+
+  actions.map(({ type, keysToRemove, extraData }) => {
```

Comment:
```suggestion
  actions.forEach(({ type, keysToRemove, extraData }) => {
```
in general here you should prefer `forEach` too, cause you don't return anything on each iteration, just do some logic